### ISSUE TITLE
Shrink library bundle size

### ISSUE THREAD

**Issue by dvago** [apexcharts/vue-apexcharts#112]
Reviewing a bundle built using Webpack via Vue-cli I've noticed the humongous size took by `ApexCharts`, The library is around twice the size of the popular library `Chart.js`

![Screenshot 2019-06-04 at 13 06 00](https://user-images.githubusercontent.com/5074033/58952243-17bd5a00-878b-11e9-8263-70943e0ddf46.png)

Would it be possible to optimise the dist size?

An idea could be giving the user the possibility to remove unused type of charts.
The same way it's possible to remove locales from `Moment.js` specifying just the local you need:
```
  new webpack.ContextReplacementPlugin(/moment[/\\]locale$/, /en/),
```

e.g. A project has just line and bar charts within the UI but the ApexCharts lib bundle contains all the other types

**Comment by junedchhipa** [comment]
Why `apexcharts.esm.js` and `apexcharts.common.js` both are loaded?
Only 1 should be loaded.

**Comment by dvago** [comment]
It's what webpack (via vue-cli) generates on build using the import method:
```
import VueApexCharts from 'vue-apexcharts'
```

```
Vue.use(VueApexCharts)
Vue.component('apexchart', VueApexCharts)
```

**Comment by henriquefurlan** [comment]
I'm facing the same problem, both `apexcharts.esm.js` and `apexcharts.common.js` are loaded.

I'm using `nuxt`, is it something related?

**Comment by dvago** [comment]
@henriquefurlan it's not dependent on `nuxt`, hopefully we'll get a feedback from some Apexchart contributor.

**Comment by junedchhipa** [comment]
I imported different files in the same component accidentally.
Just fixed it and released 1.4.0
